package.json

```diff
@@ -2,10 +2,14 @@
     "name": "ogl",
     "version": "1.0.0",
     "description": "WebGL Library",
-    "main": "./src/index.js",
-    "exports": "./src/index.js",
     "type": "module",
+    "main": "./src/index.js",
+    "exports": {
+        ".": "./src/index.js",
+        "./src/*": "./src/*"
+    },
     "sideEffects": false,
+    "types": "./types/index.d.ts",
```

Comment:
Is there any reason to include `src` here? Also note that since types aren't co-located with src, you'll need a `types` key within `exports`:

```js
".": {
   "types": "./types/index.d.ts",
  "default": "./src/index.js"
},
```

Reply:
I included `src` there following the same convention as three.js, which allows importing directly from the source files as well.

https://github.com/mrdoob/three.js/blob/2d1695ed65e39c293223a7651f78f24de265feb7/package.json#L16

Reply:
The `types` entry point works without that export, but happy to add that back if you need it. 😊

Reply:
The issue is that `exports` takes precedence over the top-level `types` and other fields.

Reply:
Ok, updated and tested. Works for me! 👍